### DISCUSSION TITLE
단어사전의 '상위 일정'에 대하여

### DISCUSSION THREAD

**Discussion by CJC0512** [OmokNoonE/PPM-backend#8] [General]
단어 사전에서 사용하고 있는 '상위 일정'은 일정의 트리 구조를 표현하기에 애매한 부분이 있습니다.

가령, 회원 기능 개발이라는 일정이 존재한다면
```
                       회원 기능
                           |
회원 가입(등록) | 로그인  | 회원 탈퇴(soft delete) 
```
이 존재할텐데, '회원 가입(등록)' 일정의 입장에서 '로그인' 일정을 표현하기 모호합니다.

'상위 일정'이란 단어사전에서 '현재 일정보다 상위에 있는 일정'으로 표현되는데,
상위 라는 표현은 1 depth 위로 해석되어 회원 기능의 같은 depth의 다른 기능으로 해석될 여지가 있습니다.

따라서, '상위 일정' 대신 '부모 일정'으로 표현하는 건 어떻게 생각하는지 의견을 여쭙고 싶습니다. 
이렇듯 가족 개념으로 일정을 표현한다면, '회원 가입(등록)'의 입장에서 '로그인' 일정을 단순히 '형제 일정'이라는 간단한 문장으로도 표현이 가능합니다.

※ 두 줄 요약
1. '상위 일정'을 '부모 일정'으로 바꾸자.
2. 일정에 가족의 개념을 적용시켜, '형제 일정', '자식 일정' 등의 개념을 단어 사전에 추가하자.

**Comment by EI-FIE** [comment]
동의합니다 :P

**Comment by jihyeongeun** [comment]
좋은 의견이라 생각합니다~
